mattermost-plugin/webapp/src/components/FocalboardUnfurl.tsx

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React, {useState, useEffect} from 'react'
+import ReactMarkdown from 'react-markdown'
+
+const Avatar = (window as any).Components.Avatar
+const Timestamp = (window as any).Components.Timestamp
+const imageURLForUser = (window as any).Components.imageURLForUser
+
+import './FocalboardUnfurl.scss'
+
+type Props = {
+    embed: {
+        type: string,
+        data: string,
+    }
+}
+
+export const FocalboardUnfurl = (props: Props): JSX.Element => {
```

Comment:
Should `props.embed.type` be validated?

Reply:
No it shouldn't. Good call cause we don't actually care about it here.